taqueria-plugin-tezos-client/README.md

```diff
@@ -21,88 +15,34 @@ To install the Tezos Client plugin on a Taqueria project, navigate to the projec
-
-The structure of the command for the typecheck task including all optional parameters is:
+Basic usage is:
 ```shell
-taq typecheck [inputFiles] -s sandboxName
+taq typecheck <filename>
 ```
 
-A particular Michelson smart contract can be specified by the optional `<inputFile>` argument. If the file name is not specified, the task will typecheck all Michelson smart contracts in the `/artifacts` and `/contracts` directories
-
-### Optional Parameters
+### Basic description
+This task typechecks a Michelson file in `/artifacts` and ouputs a result suggesting a failure or success.
 
-The `typecheck` task will accept the following optional parameters:
+## The `taq simulate` task
 
-| flag  |  name       | description                           |   
-|:-----:|:------------|---------------------------------------|
-|  -s   | sandboxName | The name of the sandbox to target.    |
-
-### The `taq simulate` task
-
-The `taq simulate` task uses the tezos-client instance running in the Flextesa sandbox to simulate calling a Michelson smart contract. You pass in the input parameters for the contract's `main` entrypoint, and specify the storage value for the contract through an optional flag or via the initial storage value set for the contract in the sandbox property of `config.json` 
-
-The structure of the command for the typecheck task including all optional parameters is:
+Basic usage is:
 ```shell
-taq simulate <inputFile> <inputData> -s sandboxName --storage valueOfInitialStorage
-```
- This command has 2 required parameters, and 2 optional parameters as shown above
- 
-### Required Parameters
-- `<inputFile>` - The name of the Michelson smart contract to simulate. The task will first look in the `/artifacts` directory, then the `/contracts` directory and execute the first instance of the contract name
-- `<inputData>` - The input data to pass to the Michelson smart contract. This is a single quote wrapped string value which is passed to the Michelson smart contract as the `input` parameter
-
-### Storage Value
-
-The storage value associated with the contract must be provided to the tezos-client and is what the simulation of the contract will use. There are two ways you can provide this value
-
-It can be provided as an optional parameter to the `taq simulate [inputFile]` command by adding the optional parameter: `--storage 'storageValue'`
-
-If not specified, the storage value will be set to the value of the `storage` property of the `config.json` file as seen here:
-```json ./.taq/config.json
-    "environment": {
-        "default": "local",
-        "sandbox": {
-            "networks": [],
-            "sandboxes": [
-                "local"
-            ],
-            "storage": {
-                "add_subtract_contract.tz": 10,
-            }
-        },
+taq simulate <filename> <--param paramFilename>
 ```
 
-### Optional Parameters
+### Basic description
```

Comment:
Should follow the style convention of plugin pages (this is great content for release notes though!)

Reply:
I have been refactoring the style convention btw (ligo and taquito for example). I feel like it's easier to structure them like this.

Reply:
The format is similar to the git docs, which works for me

My issue right now is inconsistency, but we can deal with that in an upcoming sprint